amp/amp-theme/index.html.twig

```diff
@@ -35,7 +35,7 @@ Available variables:
       {% if media.renditions is iterable %}
           {% for rendition in media.renditions %}
               {% if rendition.name == '770x515' %}
```

Comment:
Yeah.. this is actually wrong rendition size (from Liquid). Here we need 1250x600, or 610x380